l/libva-utils/stone.yaml

```diff
@@ -0,0 +1,30 @@
+#
+# SPDX-FileCopyrightText: © 2020-2025 Serpent OS Developers
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+name        : libva-utils
+version     : 2.22.0
+release     : 1
+homepage    : https://github.com/intel/libva-utils
+upstreams   :
+    - https://github.com/intel/libva-utils/archive/refs/tags/2.22.0.tar.gz : 7b57615c130427ec134bb5d2b80af516ce5bb19a40e89b1dab46a0d59760d96c
+summary     : Libva-utils is a collection of tests for VA-API (VIdeo Acceleration API)
+description : |
+    Libva-utils is a collection of tests for VA-API (VIdeo Acceleration API)
+license     : UPDATE LICENSE
```

Comment:
It's `MIT`: https://github.com/intel/libva-utils/blob/master/CONTRIBUTING.md?plain=1#L3

Reply:
Of course its not in the license file why on earth would you put it there sigh